### ISSUE TITLE
[Bug] Issue when EnableSchemaFolder and linking tables across schema

### ISSUE THREAD

**Issue by skuami** [TrackableEntities/EntityFrameworkCore.Scaffolding.Handlebars#164]
**Describe the bug**
Model classes are not recognized when EnableSchemaFolder is enabled. This occures when a constraint is configured on the database between tables in different schemas.
The namespaces for references tables in different schemas are missing:

![image](https://user-images.githubusercontent.com/13868838/114059740-f81a7f80-9894-11eb-8d9d-f3b525b6aa67.png)

There is also an issues on the context:
![image](https://user-images.githubusercontent.com/13868838/114059836-0ff20380-9895-11eb-962c-77d1f8afe4d6.png)

**To Reproduce**
I used the sample project from the git repo and created the northwind database with the provided SQL script.
In sql management studio I created a new schema called "app" and created a table TestTable (see screenshot).
Finally I linked the app.TestTable and dbo.Employee tables.

![image](https://user-images.githubusercontent.com/13868838/114060313-8abb1e80-9895-11eb-8392-f2529730b777.png)

**Make My Life Easier**
I didn't change any code on the sample app - so no PR provided. ;-)
EDIT: To make your life easier, I wrote the SQL script to create the schema, table and relation:

```
USE [NorthwindSlim]
GO

CREATE SCHEMA App;  
GO

CREATE TABLE App.TestTable
	(
	Id int NOT NULL IDENTITY (1, 1),
	Name nvarchar(50) NOT NULL,
	EmployeeId int NOT NULL
	)  ON [PRIMARY]
GO
ALTER TABLE App.TestTable ADD CONSTRAINT
	PK_TestTable PRIMARY KEY CLUSTERED 
	(
	Id
	) WITH( STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]

GO

ALTER TABLE App.TestTable ADD CONSTRAINT
	FK_TestTable_Employee FOREIGN KEY
	(
	EmployeeId
	) REFERENCES dbo.Employee
	(
	EmployeeId
	) ON UPDATE  CASCADE 
	 ON DELETE  CASCADE 

```

**Comment by tonysneed** [comment]
@skuami, Thanks for pointing this out. I will reproduce as soon as I have an opportunity. Of course, a PR with a fix would be welcome! 😄

**Comment by skuami** [comment]
@tonysneed, at the moment I am super busy in multiple projects but as soon as I find some time, I will have a look. 
My workaround for now ist not to use folders for schemas. That works perfectly fine.

**Comment by tonysneed** [comment]
This is fixed by #168.
